sktime/forecasting/base/tests/test_fh.py

```diff
@@ -578,6 +621,50 @@ def test_auto_arima():
     )
 
 
+def test_auto_arima_case_with_naive():
```

Comment:
I think this also needs the `pmdarima` soft dependency check. The same decorator from `test_auto_arima` is probably enough.

Reply:
fixed, the undecorated test should be with the `NaiveForecaster`, see below.